lib/index.ts

```diff
@@ -24,8 +24,8 @@ InvalidTokenError.prototype.name = "InvalidTokenError";
 
 function b64DecodeUnicode(str: string) {
   return decodeURIComponent(
-    atob(str).replace(/(.)/g, (m, p) => {
-      let code = p.charCodeAt(0).toString(16).toUpperCase();
+    atob(str).replace(/(.)/g, (match) => {
```

Comment:
I have removed the `p` argument here, as it seems that in all test cases the values between `m` and `p` are the same. This helps avoid an implicit any of `p` that the linter warns about.

If there are any cases where this could differ, please let me know and I will add a test for it.

Reply:
I would need to look into this.

Reply:
Yes, I am curious if there are test strings we can use to ensure this is covered properly.

Reply:
Let's revert this change. This PR is about linting config, let's not touch actual logic in here. We can reconsider if needed in a follow up PR, but let's keep the focus clear (makes reverting easier as well).

Reply:
Ok, it's been reverted, and now requires an additional type-cast. I'll make this a seperate PR so we can discuss it there when this one is merged.